python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -236,14 +278,17 @@ def _mg_call_plc_uniform_neighbor_sample(
         ddf_offsets = dask_cudf.from_delayed(
             [r[1] for r in result], meta=empty_df[1], verify_meta=False
         ).persist()
-        wait(ddf)
-        wait(ddf_offsets)
```

Comment:
I suggest keeping this as we are releasing the futures below. It can potentially cause issues .

Reply:
Suggest using wait like this. 

```python3
wait([ddf, ddf_offsets])
```